index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+    include "includes/header.php";
+    
+    //write PHP here...
+    $weight = 180;
+    $age = 48;
+    $firstName = "Scott";
+    $lastName = "Fincham";
+    $kg = "num";
+    
+Function poundsToKg ($kg){
+    global $weight, $kg;

```

Comment:
ideally - instead of $kg - which doesn't seem to be anything but a placeholder - you would bring in the weight in pounds as an argument. I like to call my arguments something unique to avoid confusion - so let's call it $w. This would leave you with something like:

Function poundstokg ($w){
    $kg = $w* (1//2.2);
    return $kg;
}

That way you avoid a global call - which I try to do unless its really needed.
Let me know if you have questions.